blueoil/datasets/pascalvoc_base.py

```diff
@@ -171,7 +171,7 @@ def _image_ids(self, data_type=None):
             return all_image_ids
 
         image_ids = [
-            image_id for image_id, gt_boxes in zip(all_image_ids, all_gt_boxes_list) if len(gt_boxes) is not 0
+            image_id for image_id, gt_boxes in zip(all_image_ids, all_gt_boxes_list) if len(gt_boxes) != 0
```

Comment:
`is not` is not allowed by newer flake8?

Reply:
no, errors `F632 use ==/!= to compare str, bytes, and int literals` are appeared

Reply:
I see.

Reply:
`is` compares identity (whether two reference points to the same object). `==` compares values.
there's no point using `is` when comparing int values.